packages/python/plotly/plotly/tests/test_optional/test_px/test_pandas_backend.py

```diff
@@ -55,3 +59,32 @@ def test_pandas_example():
     df = pd.DataFrame(np.random.randn(1000, 4), index=ts.index, columns=list("ABCD"))
     fig = df.iloc[5].plot.bar()
     assert len(fig.data) == 1
+
+
+@pytest.mark.skipif(
+    not hasattr(pd.options.plotting, "backend"),
+    reason="Currently installed pandas doesn't support plotting backends.",
+)
+def test_pandas_invalid_c_kwarg():
+    pd.options.plotting.backend = "plotly"
+    df = pd.DataFrame({"a": [1, 2, 3], "b": [4, 5, 6], "c": [7, 8, 9], "d": [1, 1, 2]})
+    with pytest.raises(
+        ValueError,
+        match="plotly.express.scatter does not support both 'color' and 'c' kwargs",
```

Comment:
I find this error message confusing: `plotly.express.scatter` doesn't support `s` or `c` at all...